walkthrough-1A/README.md

```diff
@@ -0,0 +1,60 @@
+# Prerequisites
+
+- The installation has already been performed on an OpenShift cluster as per the installation document https://github.com/integr8ly/installation/blob/master/README.md.
+
+# Setting up the Enmasse Address
```

Comment:
@JameelB Perhaps we should have a set before this about selecting OpenShift on the right-hand side as the login option when brought to the auth page for EnMasse. This seems to confuse a lot of people.

Reply:
I can change the first sentence to: `Open the Enmasse Console, click Openshift and login using your Openshift credentials` Wdyt?